subsys/bluetooth/controller/Kconfig

```diff
@@ -198,6 +198,29 @@ config BT_CTLR_RX_BUFFERS
 	  connection interval and 2M PHY, maximum 18 packets with L2CAP payload
 	  size of 1 byte can be received.
 
+config BT_CTLR_ISO_RX_BUFFERS
+	int "Number of Isochronous Rx buffers"
+	depends on BT_CTLR_SYNC_ISO || BT_CTLR_CONN_ISO
+	default 8
+	range 1 30
+	help
+	  Set the number of Isochronous Rx PDUs to be buffered in the
+	  controller. Number of required RX buffers would worst-case be
+	  the number of RX nodes prepared in one ISO event for each
+	  active ISO group. This depends on the number of bursts in an
+	  ISO group and number of groups, and may need to be set lower
+	  that the theoretical maximum. Default of 8 is for supporting
+	  two groups of 4 payloads, e.g. 2 CIGs with 2 CISes of BN=2.
+
```

Comment:
Set the range 1..30 without explanation. Didn't want to make it too specific, but as starting point given the other system defaults, with a max. BN of 15, a realistic scenario could use 30 buffers (2 CISes x 15 payloads). If this doesn't make sense and a better limit can't be found, I suggest to remove the range.

Reply:
> Default of 8 is for supporting two groups of 4 payloads, e.g. 2 CIGs with 2 CISes of BN=2.

If default it relative to another Kconfig, `BT_CTLR_CONN_ISO_GROUPS` and `BT_CTLR_CONN_ISO_STREAMS`, then the value should be relative to their respective values in my opinion. 

Default could thus be `BT_CTLR_CONN_ISO_GROUPS * BT_CTLR_CONN_ISO_STREAMS * 2` which seems to be the formula you use here. 

Another issue is that that it is tuned for a Connected ISO build, and does not consider the iso sync receiver.

Reply:
> Another issue is that that it is tuned for a Connected ISO build, and does not consider the iso sync receiver.

Yes, exactly. This is why I suggested to add the explaining text, to let it be up to the user to do the math for the applicable use case. I think it would be too complex to calculate with all variations taken into consideration. Also, there might be other deciding factors such as available memory and architecture. Using wording "group" and "payload" is intended to include all ISO RX cases.

Reply:
I won't this be a blocking factor of this PR, and I don't have any proper solutions for it either.